components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -154,6 +154,14 @@ class SystemEngineSession(
         notifyObservers { onTrackerBlockingEnabledChange(false) }
     }
 
+    /**
+     * See [EngineSession.close]
+     */
+    override fun close() {
+        super.close()
+        webView.destroy()
```

Comment:
Please explain why destroy is called here rather than as part of the lifecycle. Maybe mention destroying a WebView is okay to do here because each EngineSession holds its own WebView reference?

Reply:
Done.